src/Core/Models/Api/Request/Organizations/OrganizationSsoRequestModel.cs

```diff
@@ -5,34 +5,53 @@
 using Bit.Core.Models.Data;
 using Bit.Core.Enums;
 using Bit.Core.Sso;
-using Bit.Core.Settings;
 using U2F.Core.Utils;
 using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
+using System.Text.Json;
 using System.Text.RegularExpressions;
+using Bit.Core.Models.Table;
 using Microsoft.AspNetCore.Authentication.OpenIdConnect;
 
-namespace Bit.Portal.Models
+namespace Bit.Core.Models.Api
 {
-    public class SsoConfigDataViewModel : IValidatableObject
+    public class OrganizationSsoRequestModel
     {
-        public SsoConfigDataViewModel() { }
+        [Required]
+        public bool Enabled { get; set; }
+        [Required]
+        public SsoConfigurationDataRequest Data { get; set; }
+
+        public SsoConfig ToSsoConfig(Guid organizationId)
+        {
+            return ToSsoConfig(new SsoConfig { OrganizationId = organizationId });
+        }
+
+        public SsoConfig ToSsoConfig(SsoConfig existingConfig)
+        {
+            existingConfig.Enabled = Enabled;
+            var configurationData = Data.ToConfigurationData();
+            existingConfig.Data = JsonSerializer.Serialize(configurationData, new JsonSerializerOptions
+            {
+                PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
```

Comment:
This is not essential, but this config option is repeated frequently and is easy to miss unless you know to use it. Is it worth having a `CoreHelpers` method, like `SerializeWithCamelCase` or `SerializeForJavascriptClients` etc? I guess you've still got to know to use that `CoreHelpers` method, but it might be a bit more discoverable. (and more succinct)

Reply:
Good point, I've added an Asana task to track it. (Since we would need to refactor many areas which feels a bit out of scope for this issue.